unnamed_toaster/unnamed_toaster/launch/unnamed_toaster.launch

```diff
@@ -28,4 +28,9 @@
     <param name="ip" type="string" value="10.17.21.2" />
   </node>
 
+  <!-- Start game logic -->
+  <node name="match_logic" pkg="unnamed_toaster" type="match_logic.py" output="screen">
+    <param name="rate" type="int" value="50" />
+  </node>
```

Comment:
this is fine for now, let's make it a different upstart job for match stuff though - I'll help you with that next week

Reply:
Thank you i would appreciate that a lot :)